src/app/auth.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { BackendResolver } from '../app/Services/backendResolver.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthService {
+
+  isLoggedIn = false;
+  readonly cookieKey: string;
+
+  constructor(private readonly myRoute: Router, private readonly backend: BackendResolver) {
+    this.cookieKey = backend.cookieKey;
+  }
+
+  loggedIn(): boolean {
+    this.isLoggedIn = localStorage.getItem(this.cookieKey) != null;
```

Comment:
The backend resolver should do this work. Maybe a method like `isLoggedIn(): boolean` on BackendResolver?